api/internal/init.js

```diff
@@ -76,11 +76,20 @@ if ((globalThis.window || globalThis.self) === globalThis) {
 
 // webview only features
 if ((globalThis.window) === globalThis) {
-  globalThis.addEventListener('dragdropfiles', async (event) => {
-    const { files } = event.detail
+  globalThis.addEventListener('platformdrop', async (event) => {
```

Comment:
probably fine in this context, but this `async` func has no catch, and being passed to something that doesn't handle the promise right? ... so I think it is gonna swallow errors or trigger a confusing unhandled-promise error if something goes wrong